lib/credit_card_widget.dart

```diff
@@ -276,17 +280,21 @@ class _CreditCardWidgetState extends State<CreditCardWidget>
       height: widget.height,
       width: widget.width,
       padding: widget.padding,
+      border: widget.frontCardBorder,
       child: Column(
         crossAxisAlignment: CrossAxisAlignment.start,
         children: <Widget>[
-          if (widget.bankName != null && widget.bankName!.isNotEmpty)
-            Padding(
-              padding: const EdgeInsets.only(left: 16, top: 16),
-              child: Text(
-                widget.bankName!,
-                maxLines: 1,
-                overflow: TextOverflow.ellipsis,
-                style: defaultTextStyle,
+          if (widget.bankName?.isNotEmpty == true)
```

Comment:
You can pass directly condition without comparing it. And nullable variable troubling then add default value false